.github/workflows/cd-explore-stage.yml

```diff
@@ -6,41 +6,47 @@ on:
     - master
 
 jobs:
-  build_linux:
+  build_deploy_linux:
     runs-on: ubuntu-latest
 
     env:
-      AZURE_WEBAPP_NAME: fast-explore
-      AZURE_WEBAPP_PACKAGE_PATH: ./sites/fast-component-explorer/www
+      AZURE_WEBAPP_ACTIVE_STAGE_NAME: explore-west-app
+      AZURE_WEBAPP_PASSIVE_STAGE_NAME: explore-east-app
+      AZURE_WEBAPP_BUILD_PATH: sites/fast-component-explorer
+      AZURE_WEBAPP_DIST_PATH: sites/fast-component-explorer/www
       AZURE_WEBAPP_SLOT_NAME: stage
 
     steps:
-    - name: Azure Login
-      uses: azure/login@v1
-      with:
-        creds: ${{ secrets.AZURE_CREDENTIALS_FAST_DESIGN }}
-
     - name: Checkout Branch
       uses: actions/checkout@master
 
     - name: Install Lerna
-      run: sudo yarn global add lerna
+      run: yarn global add lerna
 
     - name: Install package dependencies / prepare workspaces
       run: yarn install --frozen-lockfile
     
-    - name: Build Web Application
-      run: | 
-        cd sites/fast-component-explorer
+    - name: Build & Prepare Web Application
+      run: |
+        cd ${{ env.AZURE_WEBAPP_BUILD_PATH }}
         yarn build:app
+        cp ../statics/package.json ../statics/server.js www 
```

Comment:
I'm not sure we should have a single package.json for these and a single server.js for all sites. I'm specifically concerned with a single package.json file as the requirements across sites are different. Some require react and others may not.

Reply:
@chrisdholt @janechu The package.json file is required to be distributed so I can install node server packages during the packaging process.  What do you suggest?  Would you rather include, this static approach within each site itself?  I would then include the commands in the scripts of the root package.json to perform install and copy during the build.  @radium-v also as we collaborated quite a bit on different solutions and he's aware of the scenario I'm covering.

Reply:
I think it's worth asking the question, will this work for a web app that controls its own routing like a React site and will it work for a static site that has `index.html` files in various folders as its routing mechanism. If it works for both then this might not be an issue.

Reply:
@janechu if you can provide more info I can answer your question. What are these sites running, please fill in [React | Static | ...] for each:

- color
- create
- explore
- app
- www
- motion

Thanks,

Reply:
I can answer a few:
- create [react]
- explore [react]
- tooling examples [static]
- fast.design [static]

Reply:
@janechu react and static sites can all run on node express server.